packages/api/lib/publishSnsMessageUtils.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const { publishSnsMessage } = require('@cumulus/aws-client/SNS');
+const { envUtils } = require('@cumulus/common');
+
+const Logger = require('@cumulus/logger');
+
+const logger = new Logger({ sender: '@cumulus/publishSnsMessageUtils' });
+
+const constructSnsMessage = (record, event) => {
+  switch (event) {
+  case 'Create': return { event, record };
+  case 'Update': return { event, record };
```

Comment:
```suggestion
  case 'Create': 
  case 'Update':
    return { event, record };
```

This is one of the nice features of switch statements